README.md

```diff
@@ -30,3 +30,10 @@ npx playwright test
 ```shell
 npm run test
```

Comment:
I think it could be good to also mention here the other modes of operation,
like which situation they can be used for please?

(e.g. the `test:trace`, `test:host` or any other useful tips for debug/devel)

Reply:
Done, thanks